src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static Random random = new Random();
+
+    public static String getRandomColor() {
```

Comment:
according to task requirements you shouldn't use static methods
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-static-methods-in-your-solution

```suggestion
    private Random random = new Random();

    public String getRandomColor() {
```